lib/r10k/forge/module_release.rb

```diff
@@ -105,7 +128,6 @@ def unpack(target_dir)
       # Remove all files created while downloading and unpacking the module.
       def cleanup
         cleanup_unpack_path
-        cleanup_download_path
```

Comment:
This could be added back for the case of things getting stopped halfway through.